rts/System/FileSystem/FileSystemInitializer.cpp

```diff
@@ -63,6 +63,16 @@ bool FileSystemInitializer::Initialize()
 
 	SetupThreadReg();
 
+	InitializeTry();
+
+	// in case of an exception, ErrorMessageBox takes care of this
```

Comment:
Wasn't this comment about line 87/97 previously so no longer makes sense after moving?